src/lib/y2partitioner/dialogs/blk_device_resize.rb

```diff
@@ -94,6 +94,7 @@ def next_handler
 
       def detect_space_info
         return unless formatted? && controller.committed_device? && !swap?
+        return unless controller.committed_current_filesystem?
         @space_info = device.filesystem.detect_space_info
```

Comment:
Moreover, to avoid similar crashes in the future, maybe we should do something like

```ruby
begin
  @space_info = device.filesystem.detect_space_info
rescue Storage::Exception
  log.error "Storage exception calling detect_space_info"
  # Should we alert the user in addition to registering the error?
  @space_info = nil
end
```

Uncatched libstorage-ng exceptions look uglier than exceptions thrown in Ruby. Moreover, displaying the widget without the "currently used" info is acceptable, so an error detecting the available space is actually not severe enough to interrupt the execution.

Maybe we could even inform the user that something weird happened checking that filesystem, in addition to just logging the error. Then is up to the user whether to try the resizing or not.

Reply:
If we cannot determine the space info then the FS is probably broken or in a bad shape. In that case resizing it might be very dangerous, it might fail or the data can be lost. In this case I'd really appreciate a warning message with "resizing it may fail or cause data loss" info.